src/dbus/common/constants.hpp

```diff
@@ -63,6 +63,7 @@
 #define OTBR_DBUS_GET_PROPERTIES_METHOD "GetProperties"
 #define OTBR_DBUS_LEAVE_NETWORK_METHOD "LeaveNetwork"
 #define OTBR_DBUS_SET_NAT64_ENABLED_METHOD "SetNat64Enabled"
+#define OTBR_DBUS_MIGRATE_METHOD "Migrate"
```

Comment:
What about calling it `ScheduleMigration` to be consistent with [Android API](https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/Connectivity/thread/framework/java/android/net/thread/ThreadNetworkController.java;l=549;drc=5fc74b9fd8a31d6703ba760fcafe9bcfb4b99b1e)?